cppapi/server/logging.cpp

```diff
@@ -114,8 +114,8 @@ void Logging::init (const std::string& ds_name, // dserver name
     // set default <rollover> threshold for RollingFileAppender
     Logging::_rft = kDefaultRollingThreshold;
     // try to get logging path from kTangoLogPathVar env. var.
-    char *ftg_path = ::getenv(kTangoLogPathVar);
-    if (ftg_path == 0 || ::strlen(ftg_path) == 0) {
+    ApiUtil::instance()->get_env_var(kTangoLogPathVar, Logging::_log_path);
+    if (Logging::_log_path.empty()) {
```

Comment:
get_env_var of ApiUtil is static, see https://github.com/tango-controls/cppTango/blob/4feffd7c8e24b51c9597a40b9ef9982dd6e99cdf/cppapi/client/ApiUtil.h#L84.

And other callers also check the return value of get_env_var so maybe something like

```
int ret = ApiUtil::get_env_var(kTangoLogPathVar, Logging::_log_path);
if(ret != 0 || Logging::_log_path.empty())
{
...
```

Reply:
Thanks! Fixed as proposed.